xml/gl.xml

```diff
@@ -79,7 +79,18 @@ typedef unsigned int GLhandleARB;
         <type>typedef void (<apientry/> *<name>GLVULKANPROCNV</name>)(void);</type>
     </types>
 
-    <!-- SECTION: GL parameter class type definitions. -->
+    <!-- SECTION: GL parameter class type definitions.
+         The groups are intended to contain all the possible legal values
+         for corresponding function parameters, but it is likely that many
+         of the groups are out of date relative to current OpenGL and OpenGL
+         ES specifications, and the many extensions to those specifications.
+         As such, they may not be a reliable source for enumeration info.
+
+         We welcome assistance from the community in making the enum groups
```

Comment:
Nit: If #292 currently (maybe) fixes everything, it might be better to say "We welcome assistance from the community in ensuring that the enum groups are complete." That is - we may not be sure that there's anything wrong, so although we should warn that they might not be, and community assistance in checking either way would be useful, we might not want to claim there *is*. It might be embarrassing for a student to make a term project in open source software out of completing these enums and then find that there was nothing to do, for example.

Reply:
There's no guarantee that any of the groupings are complete. Fred's PR only *adds* missing groups, it doesn't at all look at any of the groupings that were already in the registry (which might not contain all possible values).

I don't think the groupings should ever be implied to be complete without Khronos managing them themselves.

Reply:
Does my proposal:

> We welcome assistance from the community in ensuring that the enum groups are complete.

...imply that we think they *are*? I just didn't like us saying that they're definitely *incomplete* (which is how I read "making the enum groups more complete"), unless we want to validate this statement alongside the current state of completeness.

Is this better?

> We welcome assistance from the community in achieving and maintaining the completeness of the enum groups.

Reply:
I like this better.

Reply:
Yeah go with it.

Reply:
Done (hopefully - thread has gotten pretty dense to follow all the suggestions).